engine/class_modules/sc_mage.cpp

```diff
@@ -339,6 +339,7 @@ struct mage_t final : public player_t
     buff_t* high_voltage;
     buff_t* impetus;
     buff_t* intuition;
+    buff_t* intuition_tracker;
```

Comment:
There are some use cases for "fake" buffs like this, but for a simple counter, it would be better to just add a new field to the `state_t` object instead.